content/docs/1.0.x/install/cli-install/index.md

```diff
@@ -26,46 +26,102 @@ You can install the Keptn CLI using:
 
 ## curl
 
-```
-curl -sL https://get.keptn.sh/ | bash
-```
+You can use the `curl` command to install the Keptn CLI on Linux, WSL2, and macOS.
+Windows users need `bash`, `curl`, and `awk` installed (e.g., using Git Bash).
```

Comment:
we are recommending bash on WSL2 here, but in the other file, we explain how to do things in PowerShell. I think we can get rid of the PowerShell instructions and stick with bash for Windows

Reply:
I removed the PowerShell instructions.  Also added a link to the WSL2 documentation the first time it is mentioned (for installing Keptn CLI with curl)